lib/logasm/preprocessors/whitelist.coffee

```diff
@@ -29,11 +29,6 @@ class Whitelist
     if pointer.charAt(pointer.length - 1) == '/'
       throw Error('Pointer should not contain trailing slash')
 
-  _decodePointer: (pointer) ->
```

Comment:
Should update doc https://github.com/salemove/node-logasm#whitelist, saying that it's a subset of JSONPointer and `~` and `/` are not supported.